python/lsst/obs/base/cameraMapper.py

```diff
@@ -459,12 +459,20 @@ def getVisitInfo(datasetType, pythonType, location, dataId):
 
                             setMethods("visitInfo", bypassImpl=getVisitInfo)
 
+                            # TODO: deprecate in DM-27170, remove in DM-27177
```

Comment:
I have to say thank you here for all of your very helpful deprecation comments. These have been very useful as I've started on DM-27170.